alea/src/variance.cpp

```diff
@@ -27,17 +27,38 @@ void var_data<T,Str>::reset()
 template <typename T, typename Str>
 void var_data<T,Str>::convert_to_mean()
 {
+    // This also works for count_ == 0
```

Comment:
Hmm. What's gonna happen if `count_ == 0`?
`data_`, `data2_` and `nunbiased` will all be nan, no?

Reply:
Exactly: `data_` and `data2_`, that is, the mean and variance will be `nan`.  This is correct conceptually, since both are `0./0.`.

Reply:
Fine. I agree.
Could you please amend `operator==` to return `true` immediately if `(r1.count() == 0) && (r2.count() == 0)`?

Reply:
Oh yes, good point.

Reply:
Along the same lines, and also as Emanuel(@egull) mentioned: there are no tests for `operator==`: how do we know that it works? ;)  And we have other tests that depend on it working.
(But possibly it should be a separate issue deserving a separate PR)